AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -29,15 +47,31 @@ final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueu
                 continue
             }
             reconciliationQueues[modelName] = queue
+
+            let cancellable = queue.publisher.sink(receiveCompletion: { completed in
+                switch completed {
+                case .failure(let error):
+                    self.reconciliationQueueTopic.send(completion: .failure(error))
+                case .finished:
+                    self.reconciliationQueueTopic.send(completion: .finished)
+                }
+            }, receiveValue: { mutationEvent in
+                if case .mutationEvent(let event) = mutationEvent {
```

Comment:
I think the closure variable is a reconciliationQueueEvent or something, right? It's a bit odd to have to pattern match `.mutationEvent` on a `mutationEvent`

Reply:
Thanks for pointing this out.. an indeed, it isn't always a mutationEvent!  Perhaps it was cruft left over as i was figuring out how to model this stuff with combine.

During the refactor, i've updated this to:
```
    private func onRecieveValue(receiveValue: ModelReconciliationQueueEvent) {
        if case .mutationEvent(let event) = receiveValue {
            self.eventReconciliationQueueTopic.send(.mutationEvent(event))
        }
    }
```
I believe this should alleviate your concern.